### ISSUE TITLE
Creating a Custom Resource backed by this lambda

### ISSUE THREAD

**Issue by manosntoulias** [Geekoosh/flyway-lambda#11]
I was trying to create a custom resource that automatically invokes this lambda whenever I deploy a new migration script on s3, but it occasionally fails because the custom resource response can be greater than the 4096 Byte threshold. The [AwsSdkCall](https://docs.aws.amazon.com/cdk/api/v1/python/aws_cdk.custom_resources/AwsSdkCall.html)  in the AwsCustomResource has an `output_paths` parameter for this reason that help select parts of the response and make it smaller. I have tried `info.current` which has a fixed size (The problem as I understand comes from `info.applied` which is of variable size). After that when trying flyway_resource.get_response_field("info.current") I get an error that this doesn't exist.

I came here just to confirm what would be the correct values and focus my investigations elsewhere. So the value of  `info.current` seems correct right?

**Comment by assafkamil** [comment]
Can you please try to invoke the lambda straight instead as from the custom resource and check the output?
Do you expect the total output to be so large?

**Comment by manosntoulias** [comment]
Below is the result from the aws console:

<img width="1410" alt="Screenshot 2022-03-29 at 19 32 40" src="https://user-images.githubusercontent.com/24657401/161217121-0e5eb92a-221d-4f90-9673-76ac43e52ef4.png">

And here the result from the terminal (It was a single line and I added some end line by hand):
<img width="1410" alt="Screenshot 2022-04-01 at 10 35 33" src="https://user-images.githubusercontent.com/24657401/161217157-5e54b0e0-fec0-4a99-bb1f-9e86510a409e.png">

That is way I used "info", "info.current". I assume paths in `output_paths` must match keys in the response of the lambda.

I was wondering that I can't use them because class Response is not a map but a POJO class? (Just speculating)
